### PR TITLE
blogpost about common templates.

### PR DESCRIPTION
**Pull request by ksimon1**
**What this PR does / why we need it**:
Describes what common templates are and how to use them.

Signed-off-by: Karel Simon <ksimon@redhat.com>

### CODE REVIEW

File: _posts/2020-07-01-Common_templates.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+`$ oc patch virtualmachine rheltinyvm --type merge -p '{"spec":{"running":true}}'`
+As soon as VM starts, openshift creates a new type of object - `VirtualMachineInstance`. It has a similar name to VirtualMachine.
+
+## Via UI:
+The Kubevirt project has an official plugin in OpenShift Cluster Console Web UI. This UI supports the creation of VMS using templates and template features - flavors and workload profiles.
+
+###### To install the templates:
+
+Install OpenShift virtualization operator from `Operators` > `OperatorHub`. The operator-based deployment takes care of installing various components, including the common templates.
+
```
Comment (mazzystr) [inline _posts/2020-07-01-Common_templates.md:58]: Screenshots needed for UI topics

File: _posts/2020-07-01-Common_templates.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+###### To create VM from template:
+ To create a VM from a template, choose `Workloads` in the left panel >> press `Virtual Machine` >> press `Create Virtual Machine` blue button >> choose `New with Wizard`. Next, you have to see `Create Virtual Machine` window
+This wizard leads you through the basic setup of vm (like guest operating system, workload, flavor, ...). After vm is created you can start requested vm.
+
```
Comment (mazzystr) [inline _posts/2020-07-01-Common_templates.md:62]: Screenshots needed for UI topics

File: _posts/2020-07-01-Common_templates.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+comments: true
+title: Common-templates
+pub-date: July 01
+pub-year: 2020
+---
+
+## What is a virtual machine template?
+
+The KubeVirt project provides a set of templates (https://github.com/kubevirt/common-templates) to create VMS to handle common usage scenarios. These templates provide a combination of some key factors that could be further customized and processed to have a Virtual Machine object. With common templates you can easily start in a few minutes many VMS with predefined hardware resources (e.g. number of CPUs, requested memory, etc.). Beware, common templates work only on OpenShift. Kubernetes doesn’t have support for templates.
+
```
Comment (mazzystr) [inline _posts/2020-07-01-Common_templates.md:24]: re Beware (warning) ... Use admonitions to highlight text like for example note, info, warning, error (Check reference at https://github.com/lazee/premonition which is the plugin we use)

File: _posts/2020-07-01-Common_templates.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+###### To install the templates:
+
+Install OpenShift virtualization operator from `Operators` > `OperatorHub`. The operator-based deployment takes care of installing various components, including the common templates.
+
+###### To create VM from template:
+ To create a VM from a template, choose `Workloads` in the left panel >> press `Virtual Machine` >> press `Create Virtual Machine` blue button >> choose `New with Wizard`. Next, you have to see `Create Virtual Machine` window
+This wizard leads you through the basic setup of vm (like guest operating system, workload, flavor, ...). After vm is created you can start requested vm.
+
+Please note that, after the generation step (UI and CLI), VM objects and template objects have no relationship with each other besides the `vm.kubevirt.io/template: rhel8-server-tiny-v0.10.0` label. This means that changes in templates do not automatically affect VMS, or vice versa.
```
Comment (mazzystr) [inline _posts/2020-07-01-Common_templates.md:63]: re Please note ... Use admonitions to highlight text like for example note, info, warning, error (Check reference at https://github.com/lazee/premonition which is the plugin we use)

File: .yaspeller.json [hunk 5]
```diff
@@ -115,6 +116,7 @@
     "galleria",
     "gcp",
     "GCP",
+    "Gi",
```
Comment (codificat) [inline .yaspeller.json:119]: I believe this is an actual misspell. Use GiB instead of Gi?

Comment (mazzystr) [reply .yaspeller.json:119]: We should add disk sizes to the style guide and lay down the standard.

Comment (ksimon1) [reply .yaspeller.json:119]: In common templates we are usin `Gi` units (see here: https://github.com/kubevirt/common-templates/blob/master/generate-templates.yaml#L20 - attribute `memsize`

**Comment by ksimon1** [comment]
@mazzystr  can you please review this PR?

**Pull request merged**